src/table/handle-data.js

```diff
@@ -30,7 +29,7 @@ export default async function manageData(model, layout, pageInfo) {
 
   const columns = columnorder.map((c) => getColumnInfo(layout, c));
   const rows = matrix.map((r, rowIdx) => {
-    const row = {};
+    const row = { key: rowIdx };
```

Comment:
Using index as keys should generally be avoided. But here we are adding a key (that happens to be an index) before rendering starts, not using the index of the array when looping over the array. So this should be fine. (see https://reactjs.org/docs/lists-and-keys.html#keys)